tasks.py

```diff
@@ -0,0 +1,122 @@
+from glob import glob
+import platform
+import re
+import shutil
+import toml
+import subprocess
+from pathlib import Path
+from invoke import task, call
+
+
+def _git_root():
+    output = subprocess.check_output(["git", "rev-parse", "--show-toplevel"])
+    output = output.decode().strip()
+    return Path(output)
+
+
+GIT_ROOT = _git_root()
+PACKAGES_ROOT = GIT_ROOT / "packages"
+
+if platform.system() != "Windows":
+    ACTIVATE_PATH = GIT_ROOT / ".venv" / "bin" / "activate"
+    ACTIVATE = f"source {ACTIVATE_PATH}"
+else:
+    ACTIVATE_PATH = GIT_ROOT / ".venv" / "Scripts" / "activate"
+    ACTIVATE = f"{ACTIVATE_PATH}.bat"
```

Comment:
I believe we can remove these (and their usage) since the current meta venv context is already activated when you're issuing Poetry-based invoke commands with: `poetry run inv <cmd>` (which is the correct way to do it since _invoke_ is covered as a Poetry dep)

Is there any edge-case when we need to explicitly activate the venv first? (like a case where is not activated yet or when not running under the venv Poetry context)

Reply:
Sorry, what I meant was:
1. You run invoke commands with `poetry run inv <cmd>` in your terminal.
2. Your poetry venv gets automatically activated so it runs invoke within that environment.
3. No need to re-activate the venv in the _tasks.py_ file since the meta venv is already activated. (and any other `poetry install` of other projects happens in this already activated venv)

Usually you run once `poetry install` to create the venv then issue any other command with `poetry run ...` like:
- `poetry run inv ...`
- `poetry run python ...`
- `poetry run pip ...`
This ensures the executable runs from the venv. (and btw, on a clean env you don't usually have `inv`, you only have Python and Poetry and you start from there, that's why might be the safest to just `poetry run` everything; also you don't need to run `poetry run poetry install` because the installation part is covered in the _tasks.py_ file)

All from above (with my change) was tested on both Mac & Windows and is working just fine, that's why I was curious if is there any expected intentional behaviour from activating again explicitly the meta venv inside the _tasks.py_ file.

Reply:
Ah, I understand now. The current readme only shows using invoke like `invoke lint`. I had not thought to execute all invoke commands via `poetry run`. That would solve the issue... perhaps that's something to be added to the readme as I hadn't thought of that at all before now.

My only concern is if I write the invoke tasks assuming everyone knows to execute via `poetry run`, that's a possibly bad assumption. Perhaps I can write an error message that can tell it was not ran via `poetry run`

Reply:
@cmin764 I am updating the readme to indicate invoke should be ran via poetry, but I am looking at the `tasks.py` for the main package and I see the definition for the `lint` task executes a `poetry run` command inside of it. I tested it and it doesn't cause it to break... but I guess this begs the question what is expected with `invoke`. Do all devs expect to run `invoke` via `poetry run` or should we assume the other?

That implementation was written by @mikahanninen so perhaps he can weigh in on preference. I just need to know if I should assume inside an invoke task that someone started the task via `poetry run` or not.

Reply:
Understood, all these `'run ...` commands are for paranoia-ensuring that Invoke will use the executables from the venv, so you're right, it is expected to have developers using `inv <cmd>` directly, even outside of the venv. Hence, a paranoia inner venv activation won't hurt and be aligned with rest of the code, funny thing I added `invoke` into **main** [here](https://github.com/robocorp/rpaframework/pull/309/files#diff-b99f89fed4ef40ec0f6699d9ef63897336d08d867f2e6db306b64db7e01f78f7) as a monkey-see monkey-do from **windows**. Currently these **bold** packages have `invoke` listed in their requirements:
- meta
- **aws** (Mika)
- core
- dialogs
- **google** (Mika)
- **main** (me)
- **pdf** (me)
- recognition
- **windows** (Mika)

So definitely we should decide if is expected to develop with the Poetry-installed Invoke or with a standalone external one installed separately by the developer (or with both if we want to support that), then align all packages' requirements given that.

Reply:
Based on discussion on slack, I am assuming using external `invoke` and activating the environment for all my actions. Shouldn't hurt if you did use `poetry run invoke` though.